src/cargo/ops/registry/publish.rs

```diff
@@ -319,14 +319,12 @@ fn verify_dependencies(
     Ok(())
 }
 
-fn transmit(
+pub(crate) fn prepare_transmit(
     gctx: &GlobalContext,
     pkg: &Package,
-    tarball: &File,
-    registry: &mut Registry,
     registry_id: SourceId,
-    dry_run: bool,
-) -> CargoResult<()> {
+    require_readme_and_license: bool,
```

Comment:
Why is this `bool` in here?

Reply:
I don't remember, sorry. Removing it doesn't break any tests...